File: backend/src/main/java/com/mysite/knitly/domain/mypage/dto/FavoriteProductItem.java
package com.mysite.knitly.domain.mypage.dto;

import java.time.LocalDate;
import java.math.BigDecimal;


// 마이페이지 - 내가 찜한 상품 조회

public record FavoriteProductItem(
        Long productId,
        String productTitle,
        String sellerName,     // (판매자)
        String thumbnailUrl
) {}

File: backend/src/main/kotlin/com/mysite/knitly/domain/mypage/dto/FavoriteProductItem.kt
package com.mysite.knitly.domain.mypage.dto

// 마이페이지 - 내가 찜한 상품 조회
data class FavoriteProductItem(
    val productId: Long,
    val productTitle: String,
    val sellerName: String,
    val thumbnailUrl: String?
)
